### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Hrishi5111998**
there is typo in the contributors.md file at contributing guidelines **beginner** should be spelled like this not begginer ! thank you!

## Describe the changes you've made
there is typo in the contributors.md file at contributing guidelines **beginner** should be spelled like this not begginer ! thank you!

### Type of change
What sort of change have you made:
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### How Has This Been Tested?
< Describe_how_have_you_verified_the_changes_made >

## Checklist:
- [ ] My code follows the guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.

## Screenshots (Images)
| Before the changes | After the Changes |
| --- | --- |
| <Upload_Screenshot> | <Upload_Screenshot> |

## Recordings (Video with audio)
| Before the changes | After the Changes |
| --- | --- |
| <Upload_Recording> | <Upload_Recording> |

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -138,3 +138,5 @@ Once you've submitted your pull request, you want other members of the developme
 <div id="text" align="center">
 <h2>Happy Hacking...!</h2>
 </div>
+![2023-01-24 (1)](https://user-images.githubusercontent.com/115489262/214350275-0cbc4994-d775-438f-af44-c4c0bb611f7b.png)
```
Comment (prakhartiwari0) [inline CONTRIBUTING.md:141]: I think you have mistakenly added this image to the file, kindly remove it. @Hrishi5111998

**Review by prakhartiwari0 — CHANGES_REQUESTED** [review]
Please remove the image

**Comment by Hrishi5111998** [comment]
> Please remove the image

oh my bad! I'll do it

File: CONTRIBUTING.md (hunk 1 above)
Comment (Hrishi5111998) [reply CONTRIBUTING.md]: image removed! thank you

**Review by prakhartiwari0 — APPROVED** [review]

**Pull request merged**